src/api/java/openiss/ws/soap/endpoint/OpenISSSOAPServicePublisher.java

```diff
@@ -0,0 +1,13 @@
+package api.java.openiss.ws.soap.endpoint;
+
+import api.java.openiss.ws.soap.service.OpenISSSOAPServiceImpl;
+import javax.xml.ws.Endpoint;
+
+public class OpenISSSOAPServicePublisher {
+
+
+    public static void main(String[] args) {
+        Endpoint.publish("http://localhost:7779/ws/xmlread", new OpenISSSOAPServiceImpl());
```

Comment:
(A) please agree on with Team 10 in #44 (who use 9090) and refactor hardcoded 7779 to the same port number and refactor it from its hardcoded state to a `Config.java` that both would use for common settings and properties.

Reply:
checked (A) has been resolved